docs/index.md

```diff
@@ -103,38 +100,34 @@ argnorm [tool] -i [original_annotation.tsv] -o [annotation_result_with_aro.tsv]
 ### ARGs-OAP
 
 ```bash
-argnorm argsoap --mode reads -i examples/raw/args-oap.sarg.reads.tsv -o tmp
-argnorm argsoap --mode reads -i examples/hamronized/args-oap.sarg.reads.tsv -o tmp --hamronized
-
--------------------------------------------------------------------------------------------------------
+argnorm argsoap -i examples/raw/args-oap.sarg.reads.tsv -o outputs/raw/args-oap.sarg.reads.tsv
 
-argnorm argsoap --mode orfs -i examples/raw/args-oap.sarg.orfs.tsv -o tmp
-argnorm argsoap --mode orfs -i examples/hamronized/args-oap.sarg.orfs.tsv -o tmp --hamronized
+argnorm argsoap -i examples/hamronized/args-oap.sarg.reads.tsv -o outputs/hamronized/args-oap.sarg.reads.tsv --hamronized
 ```
 
 ### DeepARG
 
 ```bash
-argnorm deeparg -i examples/raw/deeparg.deeparg.orfs.tsv -o tmp
```

Comment:
I don't like that the examples use `tmp`, tbh. `output` was more descriptive for users